docs/pages/releases/2021-06-02.mdx

```diff
@@ -41,7 +41,7 @@ In addition to the JSON one above, we added new examples for:
 - Setting [Default Values](https://github.com/keystonejs/keystone/tree/main/examples/default-values) for fields.
 - [Extending the GraphQL Schema](https://github.com/keystonejs/keystone/tree/main/examples/extend-graphql-schema) with custom queries and mutations.
 
-We also published a tutorial that shows you [how to embed Keystone and SQLite in a Next.js app](https://keystonejs.com/tutorials/embedded-mode-with-sqlite-nextjs). The end result is an app with a queryable GraphQL endpoint based on your Keystone schema that you can run live on Vercel – for free! <Emoji symbol="🚀" alt="Rocket"/>
+We also published a tutorial that shows you [how to embed Keystone and SQLite in a Next.js app](https://keystonejs.com/tutorials/embedded-mode-with-sqlite-nextjs). The end result is an app with a queryable GraphQL endpoint based on your Keystone schema that you can run live on Vercel – for free! <Emoji symbol="🚀" alt="Rocket"/>
```

Comment:
This one as an interesting one. Found a non-standard empty space and removed it

Reply:
Thanks!